department-of-veterans-affairs/vets-website
# VA-13677: Remove COVID Status from VAMC Facilities

**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

This removes the COVID health status from the facility locator.

## Related issue(s)

Related to [#13673](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13673)

## Testing done

1. Run `content-build` with  `yarn preview`
2. Run `vets-website` with `yarn watch --env api=https://dev-api.va.gov entry=facilities`
3. Visit [this location finder page](http://localhost:3002/find-locations/?address=Springfield%2C%20IL&bounds%5B%5D%5B%5D=-90.393958&bounds%5B%5D%5B%5D=39.049018&bounds%5B%5D%5B%5D=-88.893958&bounds%5B%5D%5B%5D=40.549018&bounds%5B%5D%5B%5D%5B%5D=-90.393958&bounds%5B%5D%5B%5D%5B%5D=39.049018&bounds%5B%5D%5B%5D%5B%5D=-88.893958&bounds%5B%5D%5B%5D%5B%5D=40.549018&bounds%5B%5D%5B%5D%5B%5D%5B%5D=-90.393958&bounds%5B%5D%5B%5D%5B%5D%5B%5D=39.049018&bounds%5B%5D%5B%5D%5B%5D%5B%5D=-88.893958&bounds%5B%5D%5B%5D%5B%5D%5B%5D=40.549018&bounds%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D=-90.393958&bounds%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D=39.049018&bounds%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D=-88.893958&bounds%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D=40.549018&bounds%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D=-90.393958&bounds%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D=39.049018&bounds%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D=-88.893958&bounds%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D%5B%5D=40.549018&context=Springfield%2C%20Illinois%2C%20United%20States&facilityType=health&latitude=39.799018&longitude=-89.643958&page=1&radius=24&serviceType&bounds%5B%5D=-90.393958&bounds%5B%5D=39.049018&bounds%5B%5D=-88.893958&bounds%5B%5D=40.549018) to see if the covid status component is no longer there (see below).

## Screenshots

From above link

### Before

<img width="1023" alt="Screen Shot 2023-05-26 at 11 24 12 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/10790736/85a7b654-7252-411e-8b80-850ce2cd2960">

### After

<img width="1443" alt="Screen Shot 2023-05-30 at 2 28 13 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/10790736/039d82dc-95dd-4807-9e67-44962565cccb">

## What areas of the site does it impact?

* Location finder

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
